pages/tools/slack/emoji.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Slack emoji usage and best practices
+questions:
+  - admins-slack
+cSpell: 
+---
+
```

Comment:
Should there be a section on our current library of emoji's and how we use them culturally?

Reply:
yes! This will be added back in - I just am out of time for today!